app/src/services/mongo/index.ts

```diff
@@ -18,4 +22,20 @@ export default new ContainerModule((bind) => {
     bind(MongoDBConnection)
         .toSelf()
         .inSingletonScope();
+
+    bind<MongooseModelFactory<AlertInterface>>('AlertModelFactory')
+        .toDynamicValue(({container}: Context) => new MongooseModelFactory(
+            container.get(Mongoose),
+            'Alert',
+            alertSchema,
+        ))
+        .inSingletonScope();
+
+    bind<MongooseModelFactory<AlertDeathInterface>>('AlertDeathModelFactory')
+        .toDynamicValue(({container}: Context) => new MongooseModelFactory(
+            container.get(Mongoose),
+            'AlertDeath',
+            alertDeathSchema,
+        ))
+        .inSingletonScope();
 });
```

Comment:
Here is where all of the models will be defined and available to be injected into handlers.